index.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module 'webext-options-sync' {
+	interface DefineOptions {
```

Comment:
You need to export this.

Reply:
Yeah I had it exported in Refined GitHub and @bfred-it asked to not export it. I must have misunderstood him.

Reply:
I think I asked _you_ if it was necessary. 😅 TypeScript noob here